### ISSUE TITLE
TypeError: Cannot read property 'update' of undefined

### ISSUE THREAD

**Issue by AndyNormore** [Lepozepo/cloudinary#109]
Just installed your plugin, I want it to work so badly! But... I get an error starting Meteor.

in the root directory of your application.
W20170313-00:50:52.644(-4)? (STDERR) C:\Users\Andy\AppData\Local\.meteor\packages\meteor-tool\1.4.3_1\mt-os.windows.x86_32\dev_bundle\server-lib\node_modules\fibers\future.js:280
W20170313-00:50:52.646(-4)? (STDERR)                                            throw(ex);
W20170313-00:50:52.648(-4)? (STDERR)                                            ^
W20170313-00:50:52.649(-4)? (STDERR)
W20170313-00:50:52.650(-4)? (STDERR) TypeError: Cannot read property 'update' of undefined
W20170313-00:50:52.651(-4)? (STDERR)   at initCollectionHooks (packages\cramhead_cloudinary.js:347:21)
W20170313-00:50:52.652(-4)? (STDERR)   at packages\cramhead_cloudinary.js:382:3
W20170313-00:50:52.652(-4)? (STDERR)   at packages\cramhead_cloudinary.js:395:4
W20170313-00:50:52.653(-4)? (STDERR)   at packages\cramhead_cloudinary.js:399:4
W20170313-00:50:52.654(-4)? (STDERR)   at packages\cramhead_cloudinary.js:412:3
W20170313-00:50:52.657(-4)? (STDERR)   at C:\meteor\SkyRooms2\.meteor\local\build\programs\server\boot.js:303:34
W20170313-00:50:52.658(-4)? (STDERR)   at Array.forEach (native)
W20170313-00:50:52.664(-4)? (STDERR)   at Function._.each._.forEach (C:\Users\Andy\AppData\Local\.meteor\packages\meteor-tool\1.4.3_1\mt-os.windows.x86_32\dev_bundle\server-lib\node_modules\underscore\underscore.js:79:11)
W20170313-00:50:52.665(-4)? (STDERR)   at C:\meteor\SkyRooms2\.meteor\local\build\programs\server\boot.js:128:5
W20170313-00:50:52.668(-4)? (STDERR)   at C:\meteor\SkyRooms2\.meteor\local\build\programs\server\boot.js:352:5
W20170313-00:50:52.669(-4)? (STDERR)   at Function.run (C:\meteor\SkyRooms2\.meteor\local\build\programs\server\profile.js:510:12)
W20170313-00:50:52.675(-4)? (STDERR)   at C:\meteor\SkyRooms2\.meteor\local\build\programs\server\boot.js:351:11

**Comment by Lepozepo** [comment]
Hmm, there's something up with a file named cramhead_cloudinary.js. That file isn't in the original package, did you get a fork? (I'm updating this package for NPM as soon as I get the S3 port done)

**Comment by AndyNormore** [comment]
Ah okay I'll try cleaning out the project. Yours isnt the first repo I've installed, maybe I've got some junk left over. Will report back after cleaning.

**Comment by AndyNormore** [comment]
Yep, cleaned up packages. Looks like left over junk!

I also got messed up because of the coffee script, took a little messing around to get it working. Maybe add a non-coffee example for us noobs, eh? I've seen a few open issues that people are confused about.

But, I have successfully pushed my JohnSnow.jpg to my account. THANK YOU for this work.

**Comment by AndyNormore** [comment]
THANK YOU. I have this 100% working now. Take pride in seeing your plugin running on www.SkyRooms.IO - come on over and say hello :)

**Comment by Lepozepo** [comment]
Awesome! Best of luck ^_^ and yeah, I know there are a bunch of issues pending >_< I've been busy up the ass with Big Data issues lately lol. Definitely expect a big refactor to this project coming from my off-time though.

**Comment by AndyNormore** [comment]
I wouldn't say there are big problems. I think it's just lack of documentation. I happen to be good at hacking other peoples stuff to solve my problems, obviously, some people aren't. And those are your customers!

Meteor is SUCH a pain in the ass. They recommended using S3 and CloudFront which I had working, then to optimize that S3 bucket is like, 3x as much work! So I found Cloudinary and was like, I need this. Googled Meteor Cloudinary and bam, here you are.

There's going to be a lot more of us coming. So, I think just by adding the non coffee script version of the instructions you'll clear up a lot of stuff. I've got some console logs screaming about the Meteor Check package, but it doesn't seem to be a problem. I'm pushing your code in to my production build on Galaxy right now.

Personally, I think this package almost needs to be official. This is BY FAR the easiest way to get Meteor file management working.
